code/game/area/almayer.dm

```diff
@@ -414,59 +415,87 @@
 	icon_state = "stairs_upperdeck"
 	fake_zlevel = 1 // upperdeck
 
-// hull areas.
+// maintenance areas
+
+/area/almayer/maint
+
+//lower maintenance areas
+
+/area/almayer/maint/lower
+	name = "\improper Lower Deck Maintenance"
+	icon_state = "lowerhull"//tempory until i get a proper one.
+	fake_zlevel = 2 // lowerdeck
+
+/area/almayer/maint/lower/constr
+	name = "\improper Lower Deck Constr Site"
+
+/area/almayer/maint/lower/s_bow
+	name = "\improper Lower Deck Starboard-Bow Maintenance"
+
+/area/almayer/maint/lower/cryo_cells
+	name = "\improper Lower Deck Cryo Cells Maintenance"
+
+// hull areas
+
+/area/almayer/maint/hull
 
 // lower deck hull areas
 
-/area/almayer/hull/lower_hull
+/area/almayer/maint/hull/lower
```

Comment:
Any particular reason why we don't instead path as `/area/almayer/maint/lower/hull` So all `/area/almayer/maint/lower` are pathed together rather than making a new kind of "lower"?

Reply:
it's because i intend to make hull area act differently in the future(different ambiance, lower temperature etc...)
when being lower or upper is just changing a variable and there to make code shorter.